<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_GenerarHumanos">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Generar Humanos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_CantidadHumanosGenerar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_IngreseCantidadHumanos">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingrese Cantidad de Humanos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Pecar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pecar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_BuenasAcciones">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Buenas Acciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_IDHumanosBuscado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingrese ID del Humano a buscar:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_BuscarHumano">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_HumanoBuscado">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_PecadosFamiliaHumanoBuscado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>261</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_PecadosFamilia">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pecados de Familia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BuenasAcciones">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buenas Acciones de Familia</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_BuenasAccionesFamiliaHumanoBuscado">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Consultas Por Paises</string>
    </property>
    <widget class="QPushButton" name="btn_Top10PaisesMasPecadores">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 10 Paises Más Pecadores</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Top5PaisesMenosPecadores">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 5 Paises Menos Pecadores</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Top10PaisesConMasBuenasAcciones">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 10 Paises Con Más Buenas Acciones</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Top5PaisesMenosBuenos">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 5 Paises Menos Buenos</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_ConsultasPorPaises">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
